tests/qir/test_pytket_qir_14-QIRProfile.PYTKET.ll

```diff
@@ -18,9 +18,9 @@ entry:
   %1 = call i1* @create_creg(i64 10)
   %2 = call i1* @create_creg(i64 10)
   %3 = call i1* @create_creg(i64 9)
-  %4 = call i1* @create_creg(i64 64)
-  %5 = call i1* @create_creg(i64 64)
-  %6 = call i1* @create_creg(i64 64)
+  %4 = call i1* @create_creg(i64 8)
```

Comment:
I am a bit confused by the size of the `c"tk_SCRATCH_BITREG_0\00"` and other, they are still split, but only into 8 bits each? This looks like a bug to me?